crates/bevy_ui/src/render/mod.rs

```diff
@@ -155,6 +155,7 @@ pub fn build_ui_render(app: &mut App) {
             )
                 .chain(),
         )
+        .add_systems(RenderStartup, init_ui_pipeline)
```

Comment:
Not sure if this is a problem in practice, but doesn't this run unconditionally now?

Reply:
Nope! This runs in `build_ui_render`. This function is only called in `UiPlugin::build` **after** checking `enable_rendering` is true. That's the exact same condition as the one I removed in `UiPlugin::finish`.

Reply:
Oh yeah I missed that. It's not very useful though and we planned to remove it anyway.